controllers/argocd_metrics_controller.go

```diff
@@ -521,7 +521,9 @@ func newPrometheusRule(namespace string) *monitoringv1.PrometheusRule {
 					{
 						Alert: "ArgoCDSyncAlert",
 						Annotations: map[string]string{
-							"message": "ArgoCD application {{ $labels.name }} is out of sync",
+							"message":     "ArgoCD application {{ $labels.name }} is out of sync",
+							"summary":     "ArgoCDSyncAlert triggers when an Argo CD application is out of sync",
```

Comment:
(suggestion) shortened version
```suggestion
							"summary":     "Argo CD application is out of sync",
```